README.md

```diff
@@ -1,24 +1,105 @@
-# README
```

Comment:
- [ ] [OPTIONAL] Please personalize the[YOUR NAME] and [APP TYPE] fields in mit.md file